register/register.go

```diff
@@ -0,0 +1,25 @@
+package register
+
+import (
+	"context"
+
+	"github.com/slsa-framework/slsa-verifier/options"
+)
+
+var SLSAVerifiers = make(map[string]SLSAVerifier)
+
+type SLSAVerifier interface {
+	// Match matches a BuilderID.
```

Comment:
against the expected builder ID for the verifier

Reply:
+1